indico/modules/events/papers/models/call_for_papers.py

```diff
@@ -36,6 +36,7 @@ def __repr__(self):
 
     start_dt = EventSettingProperty(paper_reviewing_settings, 'start_dt')
     end_dt = EventSettingProperty(paper_reviewing_settings, 'end_dt')
+    announcement = EventSettingProperty(paper_reviewing_settings, 'announcement')
```

Comment:
I am also adding those in my PR where I worked on the announcement settings: https://github.com/indico/indico/pull/2776/commits/5e20878f9d5d9d64793afc88ce479368f52636c8#diff-f67bb6a77e79f4fc1c19acfaef552841R120